lib/utils/app_platform.dart

```
@@ -0,0 +1,20 @@
+import 'package:flutter/foundation.dart';
+import 'package:platform/platform.dart';
+
+String getPlatform({Platform platform = const LocalPlatform()}) {
+  if (kIsWeb) {
+    return 'Web';
+  }
+
+  /// A string (linux, macos, windows, android, ios, or fuchsia) representing the operating system.
+  final String operatingSystem = platform.operatingSystem;
+
+  switch (operatingSystem) {
+    case 'android':
+      return 'Android';
+    case 'ios':
+      return 'iOS';
+    default:
+      throw Exception('Unsupported platform $operatingSystem!');
```

Do we want to throw or to return "unknown"?

---

See: #772

---

> do we want to put the platform tag in there?

Yes we do!